packages/grafana-ui/src/components/EmptyState/EmptySearchState/EmptySearchState.story.tsx

```diff
@@ -0,0 +1,33 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { EmptySearchState } from './EmptySearchState';
+import mdx from './EmptySearchState.mdx';
+
+const meta: Meta<typeof EmptySearchState> = {
+  title: 'General/EmptyState/EmptySearchState',
+  component: EmptySearchState,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    controls: {
+      exclude: ['image'],
+    },
+  },
+  argTypes: {
+    children: {
+      type: 'string',
+    },
+  },
+};
+
+export const Basic: StoryFn<typeof EmptySearchState> = (args) => {
+  return <EmptySearchState {...args}>{args.children}</EmptySearchState>;
```

Comment:
FYIY I don't think you need the `{args.children}` - spreading `args` into props should do the trick (`<El children={<></>} />` is valid!)